Sources/Sentry/NSString+NSUIntegerValue.h

```diff
@@ -0,0 +1,17 @@
+//
+//  NSString+NSUIntegerValue.h
+//  Sentry
+//
+//  Created by Crazy凡 on 2019/3/21.
+//  Copyright © 2019 Sentry. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface NSString (NSUIntegerValue)
```

Comment:
Can you please rename the category to `SentryNSUIntegerValue` to make sure we have no collisions.

Reply:
Done